docs/about/contributing.md

```diff
@@ -41,148 +41,158 @@ We use [semantic-release](https://www.npmjs.com/package/semantic-release), which
   * refactor
   * test (when adding missing tests)
   * chore (maintain)
-* Scope: anything that specifies the scope of the commit. Can be blank or `*`
-* Subject: description of the commit. For **breaking changes** that require major version bump, add `BREAKING CHANGE` to the commit message.
+* Scope: anything that specifies the scope of the commit; can be blank, the issue number that your commit pertains to, or `*`
+* Subject: description of the commit
 
-**Examples commit messages:**
-* Bug fix: `fix: Remove extra space`
-* Breaking change: `feat(scm): Support new scm plugin. BREAKING CHANGE: github no longer works`
+**Important:** For any **breaking changes** that require a major version bump, add `BREAKING CHANGE` somewhere in the commit title or message.
+
+**Examples commit titles:**
+* For a bug fix: `fix: Remove extra space`
+* For a breaking change: `feat(scm): Support new scm plugin. BREAKING CHANGE: github no longer works`
```

Comment:
Oh is that what @minz1027 meant? What in specific do you want me to add? Is just `Please also tag your Pull Request with appropriate Github Labels` enough?